### PR TITLE
Update RholangBuildTest to have same scheduler as prod

### PR DESCRIPTION
**Pull request by KentShikama**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

Just in case the scheduler makes a difference. This is part of the genesis ceremony optimization.

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Review by ArturGajowy — APPROVED** [review]

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala [hunk 1]
```diff
@@ -23,15 +23,24 @@ import coop.rchain.crypto.{PrivateKey, PublicKey}
...
+import monix.execution.Scheduler
 import org.scalatest.{FlatSpec, Matchers}
 
 import scala.io.Source
 import scala.util.{Failure, Success, Try}
 
 class RholangBuildTest extends FlatSpec with Matchers {
 
+  val availableProcessors = java.lang.Runtime.getRuntime.availableProcessors()
+  implicit val scheduler = Scheduler.forkJoin(
```
Comment (lukasz-golebiewski) [inline casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: Why not move the one from `NodeRuntime` to some shared package and re-use here?

Comment (KentShikama) [reply casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: PTAL

File: shared/src/main/scala/coop/rchain/shared/RchainScheduler.scala [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package coop.rchain.shared
+
+import monix.execution.Scheduler
+import monix.execution.schedulers.SchedulerService
+
+object RchainScheduler {
```
Comment (lukasz-golebiewski) [inline shared/src/main/scala/coop/rchain/shared/RchainScheduler.scala]: `RChainScheduler`?

**Review by lukasz-golebiewski — APPROVED** [review]

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala [hunk 3]
```diff
@@ -23,16 +23,18 @@ import coop.rchain.crypto.{PrivateKey, PublicKey}
 import coop.rchain.crypto.codec.Base16
 import coop.rchain.crypto.signatures.Secp256k1
 import coop.rchain.rholang.interpreter.util.RevAddress
-import coop.rchain.shared.Log
-import monix.execution.Scheduler.Implicits.global
+import coop.rchain.shared.{Log, RchainScheduler, UncaughtExceptionLogger}
```
Comment (lukasz-golebiewski) [inline casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: Are all of those imports necessary?

**Pull request merged**